src/GUI/UI/Houses/Shops.cpp

```diff
@@ -124,17 +124,17 @@ static constexpr IndexedArray<ITEM_VARIATION, HOUSE_FIRST_ARMOR_SHOP, HOUSE_LAST
+    {HOUSE_ARMOR_SHOP_HARMONDALE,           { ITEM_TREASURE_LEVEL_2, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_SHIELD } }},
+    {HOUSE_ARMOR_SHOP_ERATHIA,              { ITEM_TREASURE_LEVEL_3, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_PLATE_ARMOR } }},
+    {HOUSE_ARMOR_SHOP_TULAREAN_FOREST,      { ITEM_TREASURE_LEVEL_3, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_CHAIN_ARMOR} }},
+    {HOUSE_ARMOR_SHOP_CELESTE,              { ITEM_TREASURE_LEVEL_5, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_PLATE_ARMOR,    RANDOM_ITEM_SHIELD } }},
+    {HOUSE_ARMOR_SHOP_PIT,                  { ITEM_TREASURE_LEVEL_5, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_PLATE_ARMOR,    RANDOM_ITEM_SHIELD } }},
     {HOUSE_ARMOR_SHOP_MOUNT_NIGHON,         { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR } }},
-    {HOUSE_ARMOR_SHOP_TATALIA_1,            { ITEM_TREASURE_LEVEL_3, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_SHIELD,         RANDOM_ITEM_SHIELD } }},
-    {HOUSE_ARMOR_SHOP_AVLEE,                { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_PLATE_ARMOR } }},
-    {HOUSE_ARMOR_SHOP_STONE_CITY,           { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_PLATE_ARMOR,    RANDOM_ITEM_SHIELD } }},
-    {HOUSE_ARMOR_SHOP_CASTLE_HARMONDALE,    { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR } }},
-    {HOUSE_ARMOR_SHOP_TATALIA_2,            { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR } }},
+    {HOUSE_ARMOR_SHOP_TATALIA_1,            { ITEM_TREASURE_LEVEL_3, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_SHIELD,         RANDOM_ITEM_SHIELD } }},
+    {HOUSE_ARMOR_SHOP_AVLEE,                { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_PLATE_ARMOR } }},
+    {HOUSE_ARMOR_SHOP_STONE_CITY,           { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_CHAIN_ARMOR,     RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_PLATE_ARMOR,    RANDOM_ITEM_SHIELD } }},
+    {HOUSE_ARMOR_SHOP_CASTLE_HARMONDALE,    { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_LEATHER_ARMOR,   RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_LEATHER_ARMOR,  RANDOM_ITEM_CHAIN_ARMOR} }},
+    {HOUSE_ARMOR_SHOP_TATALIA_2,            { ITEM_TREASURE_LEVEL_4, { RANDOM_ITEM_CHAIN_ARMOR,     RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_CHAIN_ARMOR,    RANDOM_ITEM_CHAIN_ARMOR} }},
```

Comment:
Hmm... did I mess up the tables when converting them to enums?

Reply:
OK I double-checked against 47086d4c7e9c66981a9169de01ed9130fff7afc6, table changes are OK.